### PR TITLE
Add new Interactive Guide

### PR DESCRIPTION
**Pull request by jhwang-ms**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/ITAdmin-readiness.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ Use the training resources to educate yourself about Teams, teach others or run
...
 | Configuring Microsoft Teams for Firstline Workers                           | Learn how to configure and deploy the right Microsoft Teams environment for your Firstline Workforce.                                                                                                                  | [Configuring Microsoft Teams for Firstline Workers guide](https://aka.ms/ConfigFLWInteractiveGuide)              |
+| Deploying Microsoft Teams Rooms                           | Learn how to create and configure an account to use with a Microsoft Teams Rooms and manage the device in Microsoft Teams admin center.                                                                                                                  | [Deploying Microsoft Teams Rooms](https://aka.ms/teamsroomslab)              |
```
Comment (JohanFreelancer9) [inline Teams/ITAdmin-readiness.md:82]:
```suggestion
| Deploying Microsoft Teams Rooms                           | Learn how to create and configure an account to use with Microsoft Teams Rooms and manage the device in Microsoft Teams admin center.                                                                                                                  | [Deploying Microsoft Teams Rooms](https://aka.ms/teamsroomslab)              |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**